modules/installation-configuration-parameters.adoc

```diff
@@ -1943,6 +1943,12 @@ Additional GCP configuration parameters are described in the following table:
 |The name of the GCP project where the installation program installs the cluster.
 |String.
 
+|platform:
+  gcp:
+    userProvisionedDNS:
+|Enables user-provisioned DNS instead of the default cluster-provisioned DNS solution. If you want to use this feature, you must set the `featureSet` parameter to `TechPreviewNoUpgrade`. Alternatively, you can set `featureSet` to `CustomNoUpgrade` and `featureGates` to `- GCPClusterHostedDNS=true`.
```

Comment:
This isn't a direct problem, but it might be worth rewording this to something like this:

```adoc
Alternatively, you can set `featureSet` to `CustomNoUpgrade` with `featureGates` set to `- GCPClusterHostedDNS=true`.
```